yarn.lock

```diff
@@ -3449,9 +3449,9 @@ core-js@^3.19.2:
   integrity sha512-1t+2a/d2lppW1gkLXx3pKPVGbBdxXAkqztvWb1EJ8oF8O2gIGiytzflNiFEehYwVK/t2ryUsGBoOFFvNx95mbg==
 
 core-js@^3.8.2:
-  version "3.29.0"
-  resolved "https://registry.yarnpkg.com/core-js/-/core-js-3.29.0.tgz#0273e142b67761058bcde5615c503c7406b572d6"
-  integrity sha512-VG23vuEisJNkGl6XQmFJd3rEG/so/CNatqeE+7uZAwTSwFeB/qaO0be8xZYUNWprJ/GIwL8aMt9cj1kvbpTZhg==
+  version "3.30.2"
```

Comment:
hey, vlade has a day off today and asked me to update here. core-js is a dependency of inplayer.js (the SDK) so that was what actually needed to be bumped here, right after we bumped core-js in the SDK and published a new release of it. I just made a commit upgrading inplayer.js so you can take a look at that